test-file.txt

```diff
@@ -1,19 +1,16 @@
 1.
-2.
 3.
 4.
 5.
-6.
+6. Line 2 was deleted, so this is now line 5
+And here's a new line on what is now line 6
 7.
 8.
 9.
 10.
 11.
 12.
-13.
-14.
-15.
-16.
+15. Lines 13 - 14 have been deleted, and so is line 16
```

Comment:
4. I see that you added some text here on what used to be line 15. Do we really need it?